### PR TITLE
fix concent acceptance tests

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

File: scripts/concent_acceptance_tests/force_report/test_messages.py [hunk 1]
```diff
@@ -143,7 +153,7 @@ def send_and_verify_received_reject(self, rrct):
         )
         rrct_rcv = frct_response.reject_report_computed_task
         rrct_rcv.verify_signature(self.requestor_pub_key)
-        self.assertEqual(rrct_rcv, rrct)
+        self.assertMessageEqual(rrct_rcv, rrct)
```
Comment (jiivan) [inline scripts/concent_acceptance_tests/force_report/test_messages.py:156]: Either `__eq__` method should be updated or deserialization of named tuple fixed.

File: scripts/concent_acceptance_tests/force_report/test_messages.py [hunk 2]
```diff
@@ -126,8 +134,10 @@ def test_reject_rct_timeout(self):
         arct_rcv = frct_response.ack_report_computed_task
         self.assertIsInstance(arct_rcv, message.tasks.AckReportComputedTask)
         arct_rcv.verify_signature(self.variant['pubkey'])
-        self.assertEqual(arct_rcv.report_computed_task,
-                         frct.report_computed_task)
+        self.assertMessageEqual(
+            arct_rcv.report_computed_task,
+            frct.report_computed_task
+        )
```
Comment (jiivan) [inline scripts/concent_acceptance_tests/force_report/test_messages.py:140]: ditto

**Review by jiivan — APPROVED** [review]

File: scripts/concent_acceptance_tests/force_report/test_messages.py (hunk 1 above)
Comment (shadeofblue) [reply scripts/concent_acceptance_tests/force_report/test_messages.py:156]: @jiivan yes, https://github.com/golemfactory/golem-messages/issues/348

File: scripts/concent_acceptance_tests/force_report/test_messages.py (hunk 2 above)
Comment (shadeofblue) [reply scripts/concent_acceptance_tests/force_report/test_messages.py:140]: and ditto: https://github.com/golemfactory/golem-messages/issues/348

**Pull request merged**